proto/import_sstpb.proto

```diff
@@ -280,3 +283,34 @@ message DuplicateDetectResponse {
     // ]
     repeated KvPair pairs = 3;
 }
+
+message ApplyRequest {
+    // The file name of the SST file.
+    string name = 2;
```

Comment:
why does it start from 2?

Reply:
and no 5?

Reply:
just like `DownloadRequest` reserved some number for future. I think maybe it's some traditions...🤔

Reply:
@3pointer no, numbers are skipped only because they were used in fields in older versions which were then removed. It's ok to give the tags 1, 2, 3, 4 here.

Reply:
Ok, fixed.